<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleSetDialog</class>
 <widget class="QDialog" name="ScaleSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="rb_50">
        <property name="text">
         <string>50%</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_150">
        <property name="text">
         <string>150%</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rb_100">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="rb_others">
        <property name="text">
         <string>others</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_20">
        <property name="text">
         <string>20%</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_200">
        <property name="text">
         <string>200%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>rb_200</zorder>
  <zorder>rb_150</zorder>
  <zorder>rb_50</zorder>
  <zorder>rb_20</zorder>
  <zorder>rb_100</zorder>
  <zorder>rb_others</zorder>
  <zorder>slider</zorder>
  <zorder>lbl</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>rb_200</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
